test/realtime_web/controllers/tenant_controller_test.exs

```diff
@@ -269,8 +270,22 @@ defmodule RealtimeWeb.TenantControllerTest do
     setup [:with_tenant]
 
     test "reload when tenant does exist", %{conn: conn, tenant: tenant} do
+      [%{settings: settings}] = tenant.extensions
+      settings = Map.put(settings, "id", tenant.external_id)
+      PostgresCdc.connect(Extensions.PostgresCdcRls, settings)
+      Process.sleep(1000)
+      {:ok, manager_pid, _} = Extensions.PostgresCdcRls.get_manager_conn(tenant.external_id)
+
+      {:ok, connect_pid} = Connect.lookup_or_start_connection(tenant.external_id)
+
+      assert Process.alive?(manager_pid)
+      assert Process.alive?(connect_pid)
+
       %{status: status} = post(conn, ~p"/api/tenants/#{tenant.external_id}/reload")
       assert status == 204
+      Process.sleep(1000)
+      refute Process.alive?(manager_pid)
```

Comment:
Instead of sleeping we could monitor and assert_receive DOWN... etc and wait just the minimum amount of time to notice the process death 💀